data/run_lammps_example.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+TEMPERATURE=300
+BOX_SIZE=1
+
+lmp < data/lammps_input_example -v STEP 10 -v T $TEMPERATURE -v S $BOX_SIZE
```

Comment:
I think the extension is missing on the input file. I think it should be "data/lammps_input_example.lammps"?

Reply:
also, the input file is in the same folder as the script: no need for the relative path "data/".